snippets/csharp/language-reference/operators/ShiftOperatorsExamples.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+
+namespace operators
+{
+    public static class ShiftOperatorsExamples
+    {
+        public static void Examples()
+        {
+            LeftShift();
+            LeftShiftByLargeCount();
+            LeftShiftAssignment();
+            RightShift();
+            RightShiftByLargeCount();
+            RightShiftAssignment();
+        }
+
+        private static void LeftShift()
+        {
+            // <SnippetLeftShift>
+            uint x = 0b_1100_1001_0000_0000_0000_0000_0001_0001;
+            uint y = x << 4;
+            Console.WriteLine(Convert.ToString(y, toBase: 2));
```

Comment:
I'd add a Console.WriteLine statement to display x immediately before the Console.WriteLine for y. That makes the effect of the left shift operation visually apparent. The same comment applies to a couple of the other examples.